.circleci/config.yml

```diff
@@ -24,54 +58,215 @@ jobs:
             # Fallback to main cache
             - docker-buildx-main
       - run:
-          command: |
-            export BUILDX_CACHE=/tmp/cache/buildx
-            sudo mkdir -p $BUILDX_CACHE && sudo chown -R circleci:circleci $BUILDX_CACHE
-            make images
+          name: Configure buildx cache
+          command: export BUILDX_CACHE=/tmp/cache/buildx
```

Comment:
I'm no CircleCI expert, but does this do anything for other steps? I thought each step was ran in a fresh shell.

Reply:
Yip, each step is a fresh shell. Making this a command (line 9) allows me to reuse it for the e2e-test (line 194) and build-and-push (line 224) without needing to repeat (and maintain) it